### DISCUSSION TITLE
cache_aes

### DISCUSSION THREAD

**Discussion by AlUlkesh** [Klace/stable-diffusion-webui-images-browser#5] [General]
I think there might be a problem with the cache_aes function.

If I'm not mistaken exif_cache is not defined before using it here?

https://github.com/Klace/stable-diffusion-webui-images-browser/blob/974c5be7228b17d4833e86ea9dbae870ad0970d0/scripts/images_history.py#L150

**Comment by spinagon** [comment]
It errors out if this line is actually executed. As far as I see it's just a copy-paste error from `cache_exif` function.

**Reply by AlUlkesh** [reply]
PR:
#8
